### ISSUE TITLE
Develop and Train the NER Model

### ISSUE THREAD

**Issue by grayJiaaoLi** [amosproj/amos2024ss08-cloud-native-llm#62]
## User story
1. As a data engineer
2. I want / need to develop and train an NER model using our collected data
3. So that the NER model can automatically recognize and tag entities in training data

## Acceptance criteria
* Annotate the dataset for the NER training
    * It may require manual work

* Train the NER model on the annotated dataset.

* Achieve a precision and recall rate of at least 80% on a validation set

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
